crates/data-generator/src/blocks/mod.rs

```diff
@@ -0,0 +1,65 @@
+use std::io::{Error as IoError, Write as IoWrite};
+
+use proc_macro2::TokenStream;
+use quote::ToTokens;
+use syn::parse2;
+use thiserror::Error;
+
+use self::model::{Block, BlockProperty};
+use self::output::{BlockConvert, BlockDefault, BlockStruct, PropertyConvert, PropertyStruct};
+
+pub(crate) use json::ParseError as JsonParseError;
```

Comment:
Please use `self::` for less ambiguity and better automatic formatting.

Reply:
```suggestion
pub use json::ParseError as JsonParseError;
```
Unnecessary encapsulation

Reply:
> Unnecessary encapsulation

Done; see tree at 8fe9b7bac381e73e761d8d831a82a03d5096b4d3:
https://github.com/rjacraft/rjacraft/blob/ab3ad367651e2a16b94fc91e1502b87481701c8a/crates/data-generator/src/blocks/mod.rs#L3